MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -13,16 +13,23 @@ protocol FeedView {
 	func display(_ viewModel: FeedViewModel)
 }
 
+protocol FeedErrorView {
+	func display(_ viewModel: FeedErrorViewModel)
+}
+
 final class FeedPresenter {
 	private let feedView: FeedView
+	private let feedErrorView: FeedErrorView
```

Comment:
The `feedErrorView` property could be named just `errorView` (like the `loadingView` doesn't need to mention `feed`).

Please, update it everywhere.

```suggestion
	private let errorView: FeedErrorView
```